style.css

```diff
@@ -0,0 +1,206 @@
+* {
+  margin: 0;
+  padding: 0;
+  font-weight: bold;
+}
+
+hr {
+  color: #DDD;
+}
+
+.main > .feed > .feed-post > .feed-profile-pic,
+.main > .feed > .feed-post > div > div,
+.main .compose-grid > div { 
+  margin: 4px;
+  border: 3px solid black;
+  border-radius: 7px;
+}
+
+body > section {
+  height: auto;
+}
+
+body {
+  min-height: 100vh;
+  display: grid;
+  grid-template-columns: 200px 1fr 300px;
+  column-gap: 10px;
+  margin-left: 100px;
+  margin-right: 100px;
+  height: auto;
+}
+
+.navigation {
+  display: grid;
+  grid-template-rows:  60px 60px 60px 60px 60px 60px 60px 60px 60px 60px 1fr 90px;
+}
+
+.navigation > div {
+  display: grid;
+  grid-auto-flow: column;
+  justify-content: left;
+  align-items: center;
+  padding: 10px;
+  border-radius: 10px;
+}
+
+.navigation > .tweet-button {
+  padding: 0;
+}
+
+.main .compose-grid > .compose-post > .post-attachments > div > img:hover,
+.navigation > div + div:hover {
+  background-color: #EEE;
+}
+
+.main .compose-grid > .compose-post > .post-attachments > div > img:active,
+.navigation > div + div:active{
+  background-color: #CCC;
+}
+
+.navigation > div + div + div + div + div + div + div + div + div + div + div {
+  z-index: -1;
+}
+.navigation > div + div + div + div + div + div + div + div + div + div + div:hover {
+  background-color: white;
+}
+.navigation > div + div + div + div + div + div + div + div + div + div + div:active {
+  background-color: white;
+}
+
+.main .compose-grid > .compose-post  .tweet-button:hover,
+.navigation .tweet-button:hover {
+  background-color: #2D92EB;
+
+}
+.main .compose-grid > .compose-post  .tweet-button:active,
+.navigation .tweet-button:active {
+  background-color: #4870AB;
+}
+
+.navigation img {
+  margin-right: 9px;
+  width: 30px;
+}
+
+.navigation > div {
+  font-weight: bold;
+}
+
+.navigation > div > div {
+  display: inline-block;
+  width: 40px;
+  height: 40px;
+  margin: 3px;
+  background-color: #4BA0EB;
+  border-radius: 4px;
+
+}
+
+.navigation > .tweet-button {
+  margin-top: 20px;
+  justify-content: center;
+  align-items: center;
+  border-radius: 40px;
+  width: 100%;
+  height: 100%;
+  justify-self: center; 
+  align-self: center;
+  background-color: #4BA0EB;
+  color: white;
+}
+
+/* #home { */
+/*   display: none; */
+/* } */
+
+.main {
+  display: grid;
+  grid-template-rows: 50px 150px 1fr;
+  min-height: 100vh;
+}
+
+.main {
+  background-color: #DDD;
+}
+
+.main > div {
+  margin: 2px;
+  background-color: white;
+}
+
+.main .compose-grid {
+  display: grid;
+  grid-template-columns: 90px 1fr;
+}
+
+.main > .compose-grid > .profile-pic {
+  height: 70px;
+  width: 70px;
+  border-radius: 50%;
+}
+
+.main .compose-grid > .compose-post {
+  display: grid;
+  grid-template-rows: 1fr 1fr;
+  border: none;
+}
+
+.main .compose-grid > .compose-post > .post-attachments {
+  display: grid;
+  grid-template-columns: 40px 40px 40px 40px 40px 1fr 200px;
```

Comment:
Here I would say is a good example to take advantage of the use of repeat() where there are the same dimensions for multiple grid elements, maybe.....